src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -488,9 +488,8 @@ public String getServiceFileName(Interface service, String packageName) {
     return getNotImplementedString("SurfaceNamer.getApiName");
   }
 
-  /** The name of the class that implements a particular proto interface. */
-  public String getFullyQualifiedApiWrapperClassName(Interface service, String packageName) {
-    return packageName + "." + getApiWrapperClassName(service);
+  public String getFullyQualifiedApiWrapperClassName(Interface interfaze, String packageName) {
+    return getNotImplementedString("SurfaceNamer.getFullyQuallifiedApiWrapperClassName");
```

Comment:
Add a TODO to support the general pattern of package + class name in NameFormatter. Also, fix spelling (Quallified).